mubench.pipeline/tasks/implementations/publish_findings.py

```diff
@@ -95,12 +95,17 @@ def _prepare_post(self, finding: Finding, detector: Detector, findings_path: str
         specialized_finding = detector.specialize_finding(findings_path, finding)
         markdown_dict = self._to_markdown_dict(specialized_finding)
 
-        try:
-            snippets = specialized_finding.get_snippets(version_compile.original_sources_path)
-        except SnippetUnavailableException as e:
-            logger.warning("No snippets added for %s", e.file)
-            logger.debug("Failed to find snippets: %s", e.exception)
-            snippets = []
+        snippets = []
+        for original_sources_path in version_compile.original_sources_paths:
```

Comment:
It seems we have this logic (finding a file in one of the sources/classes directories) in multiple places. Can we centralize and reuse it?